### PR TITLE
[Systemds 3011] Project description to reflect the present scope

### PR DESCRIPTION
**Pull request by j143**
as per the https://s.apache.org/systemds-3011-mail-discussion

agreed candidate:

```md
Apache SystemDS - An open source ML system for the end-to-end data science lifecycle
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ limitations under the License.
 
 # Apache SystemDS
 
-**Overview:** SystemDS is a versatile system for the end-to-end data science lifecycle from data integration, cleaning,
+**Overview:** SystemDS is a An open source ML system for the end-to-end data science lifecycle from data integration, cleaning,
```
Comment (corepointer) [inline README.md:22]: auto replace doing grammar mistakes here ;-)

File: docs/api/python/sources/index.rst.txt [hunk 2]
```diff
@@ -19,12 +19,12 @@
...
 .. ------------------------------------------------------------
 
-SystemDS
+Apache SystemDS
 ========
 
 This package provides a Pythonic interface for working with SystemDS.
 
-SystemDS is a versatile system for the end-to-end data science lifecycle from data integration,
+An open source ML system for the end-to-end data science lifecycle from data integration,
```
Comment (corepointer) [inline docs/api/python/sources/index.rst.txt:27]: auto replace doing grammar mistakes here too

**Review by corepointer — CHANGES_REQUESTED** [review]
Hi!

Looks good except the two minor mistakes.

hth, Mark

**Comment by j143** [comment]
Thank you, Mark! 😃 

I have made the changes requested.

**Review by corepointer — APPROVED** [review]

**Comment by corepointer** [comment]
LGTM - Thx for taking care of this.

**Pull request merged**